src/web_interface/filter.py

```diff
@@ -337,7 +341,9 @@ def user_has_role(current_user, role):
     return current_user.is_authenticated and user_has_privilege(current_user, role)
 
 
-def sort_roles_by_number_of_privileges(roles, privileges=PRIVILEGES):
+def sort_roles_by_number_of_privileges(roles, privileges=None):
+    if privileges is None:
```

Comment:
You can replace this and all other `if ... is None:` with `<var> = <var> if <var> else <default>`